scrapeCourse.py

```diff
@@ -0,0 +1,140 @@
+import os
+import json
+import getpass
+import django
+import json
+import re
+from bs4 import BeautifulSoup
+import requests
+
+def getCourseData(code, faculty):
```

Comment:
This file has inconsistent indentation with the rest of the project

Reply:
Will fix